www/app.js

```diff
@@ -5,7 +5,7 @@ const builtin = require('../bindings/node/builtin')
 const alterschema = require('../bindings/node')
 const packageJSON = require('../package.json')
 const METASCHEMAS = require('../metaschemas.json')
-
+import './main.scss'
```

Comment:
This is what the linter is complaining about. Any `import` should be at the top of the file.

Finally, if you do decide to import the CSS here, do take care of the existing CSS rules in the `Makefile` that make use of `sassc`